src/pages/Service.js

```diff
@@ -31,15 +31,15 @@ class Service extends Component {
   render() {
     const { data } = this.state;
 
-    const topicId = get(data, "theme.id", null);
-    const topicName = get(data, "theme.text", null);
+    const topicId = get(data, "topic.id", null);
+    const topicName = get(data, "topic.text", null);
     const title = get(data, "title", null);
     const steps = get(data, "content", null);
     const contentItems = get(data, "extra_content", null);
-    const phone = get(jsonFileData, "contact.phone", null);
-    const email = get(jsonFileData, "contact.email", null);
-    const address = get(jsonFileData, "contact.address", null);
-    const hours = get(jsonFileData, "contact.hours", null);
+    const phone = get(data, "contacts[0].contact.phone", null);
```

Comment:
Instead of picking off the just the first contact item, should we loop through the list and render multiple Contact components?

Reply:
Totes! I just wasn't sure because we were assuming single location/contact in the prev code. Filed cityofaustin/janis#51 to make sure it gets fixed.